test/valid-data/string-template-expression-literals/schema.json

```diff
@@ -24,12 +24,20 @@
         "ok": {
           "const": "id_ok",
           "type": "string"
+        },
+        "num": {
+          "type": "string"
+        },
+        "bool": {
+          "type": "string"
```

Comment:
should this be `'true' | 'false'`?

Reply:
Sure. Would you like me to add a special case for `BooleanType` in `extractLiterals`?

Reply:
Seems to have worked out just fine! All tests pass